plugin/src/main/kotlin/com/jraska/module/graph/assertion/tasks/GenerateModulesGraphTask.kt

```diff
@@ -7,6 +7,9 @@ import com.jraska.module.graph.assertion.GradleDependencyGraphFactory
 import org.gradle.api.DefaultTask
 import org.gradle.api.tasks.Input
 import org.gradle.api.tasks.TaskAction
+import java.io.File
+
+private const val GRAPH_VIZ_FILE_EXTENSION = ".gv"
```

Comment:
Hi, sorry for late response I was on long holidays.

First of all thanks a lot for the change.

I would have one request. I think we don't need to have the responsibility to put the `.gv` suffix to the file. I would keep that to the user.

Since I reply late I could do this change later by myself and just accept the PR if you don't have time now. Just let me know.

Reply:
No problem, I've removed the code appending the extension and pushed the modification.
Let me know if something is wrong